spec/factories/clearance.rb

```diff
@@ -4,7 +4,18 @@
   end
 
   factory :user do
-    email
+    email { generate(:email) }
     password { 'password' }
   end
 end
+
+FactoryBot.define do
+  sequence :name do |n|
+    "chicken#{n}"
```

Comment:
If you want to have some default realistic values, you could install and use the faker gem https://github.com/faker-ruby/faker

Reply:
I actually read a lot about faker. Wasn't sure if you wanted to go that route.